packages/moldova/README.md

```diff
@@ -0,0 +1,61 @@
+# 🇲🇩 Moldova
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the districts of Moldova:
+
+- Anenii Noi
+- Bălţi
+- Tighina
+- Briceni
+- Basarabeasca
+- Cahul
+- Călăraşi
+- Cimişlia
+- Criuleni
+- Căuşeni
+- Cantemir
+- Chişinău
+- Donduşeni
+- Drochia
+- Dubăsari
+- Edineţ
+- Făleşti
+- Floreşti
+- Unitatea Teritorială Autonomă Găgăuzia (UTAG)
+- Glodeni
+- Hînceşti
+- Ialoveni
+- Leova
+- Nisporeni
+- Ocniţa
+- Orhei
+- Rezina
+- Rîşcani
+- Şoldăneşti
+- Sîngerei
+- Unitatea Teritorială din Stînga Nistrului
+- Soroca
+- Străşeni
+- Ştefan Vodă
+- Taraclia
+- Teleneşti
+- Ungheni
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/moldova`
+
+### yarn
+
+`yarn add @svg-maps/moldova`
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/moldova) under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
+
+### Changes
```

Comment:
Could you list the changes made on the original map please?

Reply:
@VictorCazanave please review